views/components/SelectConfigurationForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Select from '@material-ui/core/Select'
+import MenuItem from '@material-ui/core/MenuItem'
+import FormControl from '@material-ui/core/FormControl'
+
+const SelectConfigurationForm = ({
+  configurations,
+  updatePreferences,
```

Comment:
is `updateConfiguration` a more accurate name for this handler?

```suggestion
  updateConfiguration,
```

Reply:
also, what do you think about a more generic name like `onChange` for this prop?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/129#discussion_r943915860 
I know it's weird how this was named originally, but it's pretty accurate to what it does, because when a configuration is selected to be the default, that configuration id is stored in the user's preference object. This object lives in the user's collection away from the configurations collection. 

onChange works I could update that.